packages/react-native/Libraries/AppDelegate/RCTRootViewFactory.mm

```diff
@@ -155,6 +156,8 @@ - (UIView *)viewWithModuleName:(NSString *)moduleName
     return surfaceHostingProxyRootView;
   }
 
+  RCTInitializeUIKitProxies();
```

Comment:
thanks for adding this!

Maybe we could add this here: https://github.com/facebook/react-native/blob/main/packages/react-native/React/Base/RCTBridge.mm#L514

I am changing it for bridgeless as well. Seems like more appropriate place.